charts/jenkins/templates/config.yaml

```diff
@@ -209,7 +209,14 @@ data:
     # Install missing plugins
     cp /var/jenkins_config/plugins.txt {{ .Values.master.jenkinsHome }};
     rm -rf {{ .Values.master.jenkinsRef }}/plugins/*.lock
-    /usr/local/bin/install-plugins.sh `echo $(cat {{ .Values.master.jenkinsHome }}/plugins.txt)`;
+    version () { echo "$@" | awk -F. '{ printf("%d%03d%03d%03d\n", $1,$2,$3,$4); }'; }
+
+
+    if [ -n "$(command -v jenkins-plugin-cli)" 2>/dev/null ] && [ $(version $(jenkins-plugin-cli --version)) -ge $(version "2.1.1") ]; then
+      jenkins-plugin-cli -w /usr/share/jenkins/jenkins.war --verbose -f {{ .Values.master.jenkinsHome }}/plugins.txt;
```

Comment:
Do you need verbose on?

Reply:
no, I added it when trying to figure out what was going on, but never turned it off. I'll make sure its removed next commmit